packages/editor/src/components/post-publish-panel/postpublish.js

```diff
@@ -23,6 +23,27 @@ import { decodeEntities } from '@wordpress/html-entities';
  */
 import PostScheduleLabel from '../post-schedule/label';
 
+const POSTNAME = '%postname%';
+
+/**
+ * Returns real URL of a post. PermalinkTemplate holds all necessary
+ * information for the real URL.
```

Comment:
I would update the first sentence based on my suggestion for `getFuturePostUrl`. The second sentence is IMO unnecessary.

Reply:
changed description to 
```
* Returns URL for a future post.
```